data/json/regional_map_settings.json

```diff
@@ -1074,6 +1074,7 @@
         "cs_open_sewer": 33
       },
       "shops": {
+        "deserter_city": 50,
```

Comment:
```suggestion
        "deserter_city": 1,
```

Reply:
The GLOBALLY_UNIQUE flag works in multitile_city_buildings.json if you'd prefer that